wiki/Options/en.md

```diff
@@ -1,525 +1,162 @@
-</colgroup>
-<thead>
-<tr class="header">
-<th><p>Name</p></th>
-<th><p>Effect</p></th>
-</tr>
-</thead>
-<tbody>
-<tr class="odd">
-<td><p>S<strong>n</strong>aking Sliders</p></td>
-<td><p>Slider ball gradually &quot;snake out&quot; (crawling out) from their starting point. Not to be confused as &quot;Shaking Sliders&quot;.</p></td>
-</tr>
-<tr class="even">
-<td><p>Background Video</p></td>
-<td><p>Allow background video file to be played. Can be disabled, per beatmap, in <a href="Visual_Settings" title="wikilink">Visual Settings</a>.</p></td>
-</tr>
-<tr class="odd">
-<td><p>Storyboards</p></td>
-<td><p>Allow storyboard contents to be played. Can be disabled, per beatmap, in <a href="Visual_Settings" title="wikilink">Visual Settings</a>.</p></td>
-</tr>
-<tr class="even">
-<td><p>Combo Bursts</p></td>
-<td><p>Allow &quot;combo-burst&quot; image to appear at the side of the screen when hitting a combo milestone (30,60,100,150,200,...)</p></td>
-</tr>
-<tr class="odd">
-<td><p>Hit Lighting</p></td>
-<td><p>A subtle glow behind each hit explosions. Does not disable lighting during Kial Time.</p></td>
-</tr>
-<tr class="even">
-<td><p>Shader Effects</p></td>
-<td><p>Graphically speaking, Concert-type effects.</p></td>
-</tr>
-<tr class="odd">
-<td><p>Softening filter</p></td>
-<td><p>Adjust &quot;Shader Effects&quot; to be less flashy.<br />
-Automatically disabled if <em>Shader Effects</em> is disabled.</p></td>
-</tr>
-<tr class="even">
-<td><p>Screenshot Format (Graphics format of screenshots).</p></td>
-<td><p><strong>PNG (Lossless):</strong> Lossless, but bigger file size.</p>
-<hr>
-<p><strong>JPEG (Web-friendly) [Default]:</strong> Small file size, loads faster but &quot;lossy&quot;. May look pixelated when zoomed in.</p></td>
-</tr>
-</tbody>
-</table>
+| Name              | Effect                                                                                        |
+|-------------------|-----------------------------------------------------------------------------------------------|
+| Snaking Sliders   | Sliders will "snake out" from their starting position.                                        |
+| Background Video  | Allow background video files to play while playing beatmaps (can be disabled per-beatmap).    |
+| Storyboards       | Allow beatmap storyboards to be played (can be disabled per-beatmap).                         |
+| Combo Bursts      | Allow "combo burst" images to appear at the side of the screen upon hitting combo milestones. |
+| Hit Lighting      | A subtle glow behind each hit explosion. Does not disable lighting during Kial Time.          |
+| Shader Effects    | Graphically speaking, concert-type effects.                                                   |
+| Softening filter  | Adjust "shader effects" to be less flashy. "Shader effects" must also be enabled.             |
+| Screenshot Format | PNG (Lossless)<br>JPEG (web-friendly)(Default)                                                |
```

Comment:
change `PNG (Lossless)<br>JPEG (web-friendly)(Default)` to
`Set the screenshot file format to PNG (Lossless) or JPEG (web-friendly) (Default)`